Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        attacker_source_ips = list(set(related_alerts_df['AttackerSourceIP'].values))
</pre><p>Maybe put this block of code #41 - #59 into a function. Would make the thing a bit more contained/readable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>done!</p>